src/components/Species/index.tsx

```diff
@@ -455,7 +454,7 @@ export default function SpeciesList({ reloadData, species }: SpeciesListProps):
           searchResults?.forEach((result) => {
             speciesResults.push({
               id: result.id as number,
-              problems: result.problems as SpeciesProblemElement[],
+              problems: species.find((sp) => sp.id.toString() === (result.id as string))?.problems,
```

Comment:
What was happening here? Were there multiple unrelated problems getting attached to a species?

Reply:
No, this never worked before because the search api doesn't include the problems in the response, so problems was always undefined. It used to work with not filtering, because we used the species array that comes from App (in the props), but when filtering an calling search it was not working. Now, we removed that difference from filtering or not, and use search api always, so we were not seeing the problems at any moment

Reply:
Ah makes sense. Thank you